azurerm/resource_arm_application_gateway.go

```diff
@@ -115,6 +115,11 @@ func resourceArmApplicationGateway() *schema.Resource {
 							Required: true,
 						},
 
+						"override_path": {
```

Comment:
whilst this is named `override backend path` in the Azure Portal - it's named `path` in the API and Documentation; as such can we rename this to `path`?

Reply:
haha I second guessed myself and changed it to overrid path at the end!

Have changed this to "path" and pushed the changes